examples/consul-image/README.md

```diff
@@ -63,17 +63,14 @@ Your code should look more like this:
   "provisioners": [{
     "type": "shell",
     "inline": [
-      "git clone --branch <MODULE_VERSION> https://github.com/gruntwork-io/consul-gcp-module.git /tmp/consul-gcp-module",
-      "/tmp/consul-gcp-module/modules/install-consul/install-consul --version {{user `consul_version`}}",
-      "/tmp/consul-gcp-module/modules/install-dnsmasq/install-dnsmasq"
+        "git clone --branch `<MODULE_VERSION>` https://github.com/hashicorp/terraform-google-consul.git  /tmp/terraform-google-consul",
+        "/tmp/terraform-google-consul/modules/install-consul/install-consul --version {{user `consul_version`}}",
+        "/tmp/terraform-google-consul/modules/install-dnsmasq/install-dnsmasq"
     ],
     "pause_before": "30s"
   }]
 }
 ```
 
-You should replace `<MODULE_VERSION>` in the code above with the version of this Module that you want to use (see
-the [Releases Page](https://github.com/hashicorp/terraform-google-consul/tree/master/releases) for all available versions). That's because for production usage, you should always
-use a fixed, known version of this Module, downloaded from the official Git repo. On the other hand, when you're 
-just experimenting with the Module, it's OK to use a local checkout of the Module, uploaded from your own 
-computer.
\ No newline at end of file
+You should replace `<MODULE_VERSION>` in the code above with the version tag of this Module that you want to use. That's because for production usage, you should always use a fixed, known version of this Module, downloaded from the official Git repo. On the other hand, when you're just experimenting with the Module, it's OK to use a local checkout of the Module, uploaded from your own
+computer.
```

Comment:
I think the [Releases Page](https://github.com/hashicorp/terraform-google-consul/releases) is worth including so users know what we mean by a "version tag", but I see now that we had a typo in the original URL. Can you re-add the URL?